### ISSUE TITLE
ServerConfig class should be public API

### ISSUE THREAD

**Issue by thatsIch** [bbottema/simple-java-mail#136]
We require some post-validation after the mailer and email is constructed to check if all the required fields were set but calling

```java
public class Mailer {
    ...
    public ServerConfig getServerConfig() {
    ...
}
```

returns a package private `org.simplejavamail.mailer.ServerConfig` which is also not declared as internal via package structure.

**Comment by bbottema** [comment]
Yikes, that slipped the cracks. Thanks for the heads up. Released in Maven Central 5.0.3

**Comment by thatsIch** [comment]
Thanks for the quick fix!

Have a nice day :)
